test/Microsoft.IdentityModel.Protocols.WsFederation.Tests/WsFederationMessageTests.cs

```diff
@@ -279,12 +185,54 @@ public static TheoryData<WsFederationMessageTheoryData> GetTokenTheoryData
                                 Wresult = ReferenceXml.WResult_Saml2_Valid_With_Spaces
                             }
                         },
+                        Token = ReferenceXml.Token_Saml2_Valid,
                         TestId = "WsFederationMessage getToken test with white spaces"
+                    },
+                    new WsFederationMessageTheoryData
+                    {
+                        WsFederationMessageTestSet = new WsFederationMessageTestSet
+                        {
+                            WsFederationMessage = new WsFederationMessage
+                            {
+                                Wresult = ReferenceXml.WResult_Dummy_WsTrust1_3
+                            }
+                        },
+                        Token = ReferenceXml.Token_Dummy,
+                        TestId = "WsFederationMessage getToken test with WsTrust 1.3"
+                    },
+                    new WsFederationMessageTheoryData
+                    {
+                        WsFederationMessageTestSet = new WsFederationMessageTestSet
+                        {
+                            WsFederationMessage = new WsFederationMessage
+                            {
+                                Wresult = ReferenceXml.WResult_Dummy_WsTrust1_4
+                            }
+                        },
+                        Token = ReferenceXml.Token_Dummy,
+                        TestId = "WsFederationMessage getToken test with WsTrust 1.4"
                     }
                 };
             }
         }
 
+#pragma warning disable CS3016 // Arrays as attribute arguments is not CLS-compliant
```

Comment:
We can have a single #pragma clause set. One at the beginning of the file, closing at the end.